.circleci/deploy-config.yml

```diff
@@ -4,9 +4,14 @@ version: 2.1
 orbs:
   node: circleci/node@5.2.0
   slack: circleci/slack@4.13.3
-  aws-cli: circleci/aws-cli@4.1.3
+  aws-cli: circleci/aws-cli@5.4.0
 
 
+commands:
+  install_uv:
+    steps:
+      - run: pip install --upgrade uv">=0.5.12,<0.6.0"
+
```

Comment:
Instead of having this `install_uv` step here, can we just use `deployscripts/install_uv.sh` everywhere, like we have done in EE. Then there is only one place we need to change that constraint when we upgrade. The other apps where I did an `install_uv` CI step don't have an `install_uv.sh` script